addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -729,7 +729,7 @@
                                 <i class='fa fa-heart' /> Tip
                             </div>
                         </t>
-                        <t t-if='widget.pos.config.iface_cashdrawer'>
+                        <t t-if='widget.pos.config.iface_cashdrawer' id="pos_payment_cashdrawer">
```

Comment:
maybe not needed if we only use 'iface_cashdrawer' and if needed, use class instead of id